netio-exporter.py

```diff
@@ -134,9 +134,11 @@ def process_info(self):
             'outputs': str(self.data.get('Agent', {}).get('NumOutputs')),
             # in cobra, there is a `mac` field instead of the `SerialNumber` field
```

Comment:
```suggestion
            # in cobra, there is a `mac` field instead of the `SerialNumber` field. There are also some 4Cs, that have `SerialNumber` empty.
```